events/imperia_frankfurt_parliament_events.txt

```diff
@@ -0,0 +1,1105 @@
+﻿namespace = frankfurt_parliament
+# Vote for your favourite German prince
+frankfurt_parliament.1 = {
+	type = country_event
+	placement = scope:frankfurt_location_scope
+	title = frankfurt_parliament.1.t
+	desc = frankfurt_parliament.1.d
+	flavor = frankfurt_parliament.1.f
+	icon = "gfx/interface/icons/event_icons/waving_flag.dds"
+	event_image = {
+		texture = "gfx/event_pictures/frankfurt_parliament.dds"
+	}
+	duration = 1
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	immediate = {
+		random_state = {
+			limit = {
+				imperia_is_frankfurt_city_state = yes
+			}
+			save_scope_as = frankfurt_location_scope
+		}
```

Comment:
I'm pretty sure script_values are just numbers. I'm not aware of them being able to be used as scopes

Reply:
Yeay, it seems just a bit silly to go through every state to get this one.

Reply:
So it would need to be an effect setting the scope, if anything.

Reply:
> So it would need to be an effect setting the scope, if anything.

You just gave me an idea, what if I daisy-chained on the province ID, so, p:foobar.state and p:foobar.owner in the effect, perhaps?

Reply:
don't know if that will work.

Reply:
No it wont, just checked the actual use-cases of the current frankfurt_state triggers.

I mean.. We can do an on_monthly that updates a global variable and points it at Frankfurt state?

Reply:
nah, not worth it I think